netlogo-gui/src/main/lab/gui/Supervisor.scala

```diff
@@ -12,6 +12,7 @@ import org.nlogo.nvm.{EngineException, Workspace}
 import org.nlogo.workspace.{CurrentModelOpener, WorkspaceFactory}
 import org.nlogo.nvm.LabInterface.ProgressListener
 import org.nlogo.api.LogoException
+import java.io.{IOException}
```

Comment:
We've switched to adding a space after the `{` and before the `}`, so `import java.io.{ IOException }`.  Also, prefer to move this above the internal `org.nlogo` imports, but below the package statement.

Reply:
I should add, I know the other imports don't exactly match this style as they are older.  I'll leave it to you if you want to correct them, but we do add the spaces in new code.